test/draft_order_test.rb

```diff
@@ -0,0 +1,92 @@
+require 'test_helper'
+
+class DraftOrderTest < Test::Unit::TestCase
+  def setup
+    super
+
+    fake 'draft_orders/517119332', body: load_fixture('draft_order')
+
+    @draft_order = ShopifyAPI::DraftOrder.find(517119332)
+  end
+
+  def test_get_draft_order
+    fake 'draft_orders/517119332', method: :get, status: 200, body: load_fixture('draft_order')
+
+    draft_order = ShopifyAPI::DraftOrder.find(517119332)
+
+    assert_equal 517119332, draft_order.id
+  end
+
+  def test_get_all_draft_orders
+    fake 'draft_orders', method: :get, status: 200, body: load_fixture('draft_orders')
+
+    draft_orders = ShopifyAPI::DraftOrder.all
+
+    assert_equal 1, draft_orders.length
+    assert_equal 517119332, draft_orders.first.id
+  end
+
+  def test_create_draft_order
+    fake 'draft_orders', method: :post, status: 201, body: load_fixture('draft_order')
+
+    draft_order = ShopifyAPI::DraftOrder.create(line_items: [{ quantity: 1, variant_id: 39072856 }])
+
+    assert_equal '{"draft_order":{"line_items":[{"quantity":1,"variant_id":39072856}]}}', FakeWeb.last_request.body
+    assert_equal 39072856, draft_order.line_items.first.variant_id
+  end
```

Comment:
Can you add a test that does create or update with a 202 response? To ensure this gem still processes the response correctly?